homework01_sergio/vector.h

```diff
@@ -0,0 +1,30 @@
+class Vector {
+public:
+
+    Vector(int x, int y)
+        : mX(x)
+        , mY(y)
+    {}
+
+    int GetX() const {
+        return mX;
+    }
+
+    int GetY() const {
+        return mY;
+    }
+
+    void Add(const Vector& newVector) {
+        mX += newVector.mX;
+        mY += newVector.mY;
+    }
+
+    bool operator==(Vector newVector) {
+        return (mX == newVector.mX)
+               && (mY == newVector.mY);
```

Comment:
This could, and should be on one line - rule of thumb is: only one liners are accepted  in header files, the rest goes to the cc